app/cdap/components/Snackbar/styles.ts

```diff
@@ -13,33 +13,34 @@
  * License for the specific language governing permissions and limitations under
  * the License.
  */
+
 import { makeStyles } from '@material-ui/core';
-import { blue } from '@material-ui/core/colors';
-import { grey } from '@material-ui/core/colors';
+import { green } from '@material-ui/core/colors';
+import { red } from '@material-ui/core/colors';
 
-export const useStyles = makeStyles((theme) => ({
-  warningIcon: {
-    color: '#E97567',
-    fontSize: '20px !important',
-  },
-  errorHead: {
-    color: '#E97567',
-    fontSize: '20px !important',
+export const useStyles = makeStyles({
+  error: {
+    borderRadius: 4,
+    width: '100%',
+    top: '48px !important',
+    backgroundColor: red[600],
+    padding: '15px 18px 14px 18px',
+    display: 'block',
+    minHeight: 76,
   },
-  dismissSpan: {
+  success: {
+    width: '100%',
+    top: '48px !important',
+    backgroundColor: green[600],
+    padding: '15px 18px 14px 18px',
     display: 'block',
-    fontSize: '14px',
-    color: blue[500],
-    cursor: 'pointer',
+    borderRadius: 4,
+    minHeight: 76,
   },
-  errorMessage: {
-    color: grey[900],
-    fontSize: '14px',
-    padding: '10px',
+  MUIanchor: {
+    left: 0,
   },
-  headFlex: {
-    display: 'flex',
-    justifyContent: 'space-between',
-    alignItems: 'center',
+  MUIRoot: {
```

Comment:
dont use MUI string in any user defined classnames

Reply:
Done